cmd/soroban-rpc/internal/db/sqlmigrations/03_events.sql

```diff
@@ -1,16 +1,23 @@
 -- +migrate Up
 
 -- indexing table to find events in ledgers by contract_id
-CREATE TABLE events(
+CREATE TABLE events
+(
     id                TEXT PRIMARY KEY,
-    ledger_sequence   INTEGER NOT NULL,
-    application_order INTEGER NOT NULL,
-    contract_id       BLOB,
-    event_type        INTEGER NOT NULL
+    contract_id       BLOB(32),
+    event_type        INTEGER NOT NULL,
+    event_data        BLOB    NOT NULL,
+    ledger_close_time INTEGER NOT NULL,
+    transaction_hash  BLOB(32),
+    topic1            BLOB,
+    topic2            BLOB,
+    topic3            BLOB,
+    topic4            BLOB
 );
 
-CREATE INDEX idx_ledger_sequence ON events(ledger_sequence);
-CREATE INDEX idx_contract_id ON events(contract_id);
+CREATE INDEX idx_contract_id ON events (contract_id);
+CREATE INDEX idx_topic1 ON events (topic1);
```

Comment:
Wait why is this a delta instead of a new file? Shouldn't it be its own migration file?

Reply:
It is delta because schema is already been define in base branch` 'events-db-backend'.` It is going to be a new file when we merge to `main`

Reply:
Ah duh!